lib/streamlit/elements/checkbox.py

```diff
@@ -47,7 +48,7 @@ def checkbox(
         value : bool
             Preselect the checkbox when it first renders. This will be
             cast to bool internally.
-        key : str
+        key : str or int
             An optional string to use as the unique key for the widget.
```

Comment:
@AnOctopus I think docstring should be also updated to say `An optional string or integer to use ...` here and in other widget docstrings too

Reply:
Yep, they definitely should. I'll update them before merging